packages/squiggle-lang/src/rescript/Reducer/Reducer_ErrorValue.res

```diff
@@ -20,6 +21,7 @@ let errorToString = err =>
   | REAssignmentExpected => "Assignment expected"
   | REExpressionExpected => "Expression expected"
   | REFunctionExpected(msg) => `Function expected: ${msg}`
+  | REDistributionError(err) => `Math Error: ${DistributionTypes.Error.toString(err)}`
```

Comment:
(see my other comment) Maybe this is actually two things: `REDistributionInputError` to capture input validations (i.e. when `SymbolicDist.${kind}.make` does not return `Ok`) 

Plausible to me that such a thing would be more substantial than what we'd want to piggyback, and that it should be it's own story, tho.

Reply:
Haha, probably it's own story. This is already much too big

Reply:
I like the idea of splitting if this is the case, but also would recommend just doing that later.